### ISSUE TITLE
--range-perfdata does nothing

### ISSUE THREAD

**Issue by btassite** [centreon/centreon-plugins#565]
I'm trying to scale CPU graphs but they always scale to 100%, presumably because the ESX plugin reports the minimum and maximum thresholds by default. Appending --range-perfdata=n (where n is 0, 1 or 2) doesn't change anything though:

./centreon_plugins.pl --plugin=apps::vmware::connector::plugin --custommode=connector --connector-hostname=127.0.0.1 --mode=cpu-host --esx-hostname='HostX' --filter-perfdata=cpu0

OK: 'HostX' Total Average CPU usage '13.40%' on last 5 min | 'cpu0'=5.74%;;;0;100

Same output with --range-perfdata=n appended to the check.

--version reports:
Plugin Version: 0.1

Centreon Web version: 2.7.7
Centreon Engine version: 1.5.1
Centreon Broker version: 2.11.5
OS: CentOS: 6.8

**Comment by garnier-quentin** [comment]
I don't understand what do you want to do. --range-perfdata is for warning and critical threshold (in perfdata). Not for min/max.

**Comment by btassite** [comment]
The help text is a bit hard to parse:

```
    --range-perfdata
            Change perfdata range thresholds display: 1 = start value equals
            to '0' is removed, 2 = threshold range is not display.
```

But this gave me the impression that --perfdata-range=2 would output:

`OK: 'HostX' Total Average CPU usage '13.40%' on last 5 min | 'cpu0'=5.74%;;;;`

or something similar. with ";;;0;100" Centreon always scales the graph from 0 to 100%, with the range omitted it would leave scaling up to the graph template (where I specify Scaling and no max value, so in the above example the graph would be scaled to something like 15% intead of 100%, making smaller differences more visible. When the check returns a larger value, say 58%, the graph is scaled accordingly (e.g. to 60%), scaled back again when smaller values are measured and the time period doesn't include larger ones etc.).

**Comment by garnier-quentin** [comment]
Not it's not doing that.

**Comment by btassite** [comment]
I couldn't find other documentation on it, can you summarize what it does?
